src/docker/shared.rs

```diff
@@ -339,29 +339,61 @@ pub(crate) fn docker_mount(
     Ok(mount_volumes)
 }
 
+#[cfg(target_os = "windows")]
+fn winpath(path: &Path, verbose: bool) -> Result<PathBuf> {
```

Comment:
I believe this was added to this commit by mistake?

Reply:
no, see the previous error in https://github.com/cross-rs/cross/runs/7045847730?check_suite_focus=true